BossMod/BossModule/AIHintsBuilder.cs

```diff
@@ -37,16 +41,20 @@ public void Update(AIHints hints, int playerSlot)
             hints.FillPotentialTargets(_ws, playerAssignment == PartyRolesConfig.Assignment.MT || playerAssignment == PartyRolesConfig.Assignment.OT && !_ws.Party.WithoutSlot().Any(p => p != player && p.Role == Role.Tank));
             hints.RecommendedRangeToTarget = player.Role is Role.Melee or Role.Tank ? 3 : 25;
             if (activeModule != null)
-                activeModule.CalculateAIHints(playerSlot, player, playerAssignment, hints);
+                activeModule.CalculateAIHints(playerSlot, player, playerAssignment, hints, maxCastTime);
             else
+            {
                 CalculateAutoHints(hints, player);
+                _qb.CurrentModule?.AddAIHints(player, hints, maxCastTime);
```

Comment:
Do we still want autohints if quest module is active?

Reply:
Probably we do...

Reply:
yes, i don't want to write out forbidden zones for all the trash mob AOEs. it's much easier to use autohints and manually add whatever aoes are needed based on cast events